wifi-subsys/components/tools/src/serialization.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ *
+ * @brief       Cbor encoder and decoder functions to manipulate a group of data.
```

Comment:
```suggestion
/**
 * @ingroup storage
 * @{
 * @brief   Non Volatile Storage
 * @author  luisan00 <luisan00@hotmail.com>
 */
```

wrong format, please check the correct format